app/support/auto_logout.rb

```diff
@@ -23,6 +23,8 @@ def complete_status
   end
 
   def complete_reservation(od)
+    deactivate_relay = od.reservation.can_switch_instrument_off? && od.reservation.other_reservations_using_relay.count == 0
```

Comment:
I was initially thinking take both halves of the condition, but now I'm thinking just the `Reservation#other_reservation_using_relay?` into it's own method.